# -*- conf-mode -*-
#
# NOTE: 2021-09-20 pip install cloudflare==2.8.15 fails with 
#  AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'
# downgarding setuptools 58.0.4 to 53.0.0 fixes issue
#
#setuptools==53.0.0         # Require this first, to prevent later errors
#bs4			# 4.1.3 was installed
beautifulsoup4
celery==5.5.3
cloudflare==4.3.1
cryptography
Django>=4.1,<5.0
django-appconf		# 0.6 was installed
django-bootstrap5
django-csp==4.0
django-celery-beat==2.8.1
django-debug-toolbar>=1.7
django-environ
django-referrer-policy>=1.0
django-widget-tweaks>=1.3
elasticsearch>=7.0.0,<8.0.0
elasticsearch-dsl>=7.0.0,<8.0.0
factory-boy>=2.6.0
flower>=1.0.0
gunicorn==23.0.0
html5lib
jedi==0.19.2              # 0.18 has bug with ipython
jsonschema[format]>=4.2.1
lxml==5.1.0               # 5.2.0 lxml.html.clean module now separate
mailmanclient==3.3.5
mock>=2.0.0
mozilla-django-oidc==4.0.1
passlib
psycopg==3.2.9
pymemcache==4.0.0
python-json-logger>=2.0.7
pyopenssl
pyquery
pytest
pytest-cov>=2.5.1
pytest-flakes
pytest-pycodestyle
pytest-django>=2.9.1
python-dateutil>=2.2
scout-apm>=2.23.0
selenium
six
